app/views/main/index.html.erb

```diff
@@ -9,11 +9,18 @@
   </div>
 </div>
 
-<% unless admin? == false || admin? == nil %>
-  <div>
-    <div class="admin-col admin-box">
-      <div id="content-container" class="h2"><h2>Administrators:</h2></div>
-      <li><%= link_to "Admin Dashboard", admin_root_path, class: "btn btn-gold"%></li>
+<div>
+  <% unless admin? == false || admin? == nil %>
+    <div>
+      <div class="admin-col admin-box">
+        <div id="content-container" class="h2"><h2>Administrators:</h2></div>
+        <li><%= link_to "Admin Dashboard", admin_root_path, class: "btn btn-gold"%></li>
+      </div>
     </div>
+  <% end %>
+
+  <div class="admin-col admin-box">
+    <div id="content-container" class="h2"><h2>Want to explore our Committee Member Graph?</h2></div>
+    <li><%= link_to "Committee Member Dashboard", committee_member_dashboard_path, class: "btn btn-main" %></li>
```

Comment:
I think for now let's get rid of this.  We should probably be displaying a link to the committee member graph from somewhere here: https://etda.libraries.psu.edu/, since that's the more public facing side of the ETDA system.